File: specification/resources/resource-manager/Microsoft.Features/stable/2021-07-01/examples-js/getFeature.js
const { FeatureClient } = require("@azure/arm-features");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets the preview feature with the specified name.
 *
 * @summary Gets the preview feature with the specified name.
 * x-ms-original-file: specification/resources/resource-manager/Microsoft.Features/stable/2021-07-01/examples/getFeature.json
 */
async function getFeature() {
  const subscriptionId = "subid";
  const resourceProviderNamespace = "Resource Provider Namespace";
  const featureName = "feature";
  const credential = new DefaultAzureCredential();
  const client = new FeatureClient(credential, subscriptionId);
  const result = await client.features.get(resourceProviderNamespace, featureName);
  console.log(result);
}

getFeature().catch(console.error);

File: specification/resources/resource-manager/Microsoft.Features/stable/2021-07-01/examples-java/getFeature.java

/** Samples for Features Get. */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/resources/resource-manager/Microsoft.Features/stable/2021-07-01/examples/getFeature.json
     */
    /**
     * Sample code: Get feature.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void getFeature(com.azure.resourcemanager.AzureResourceManager azure) {
        azure.genericResources().manager().featureClient().getFeatures().getWithResponse("Resource Provider Namespace",
            "feature", com.azure.core.util.Context.NONE);
    }
}
